Week5/Blockbuster/app/src/main/java/com/example/blockbuster/fragments/LoginFragment.kt

```diff
@@ -0,0 +1,47 @@
+package com.example.blockbuster.fragments
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import androidx.navigation.findNavController
+import com.example.blockbuster.R
+import com.example.blockbuster.data.DataManager
+import kotlinx.android.synthetic.main.fragment_login.*
+
+
+class LoginFragment : Fragment() {
+    private  lateinit var  model : DataManager
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        model = activity?.let { ViewModelProvider(it).get(DataManager::class.java) }!!
+    }
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // -------------------------------------Hide toolbar
+        val activity = activity as AppCompatActivity
+        activity.supportActionBar?.hide()
+        // -------------------------------------------------
+        return inflater.inflate(R.layout.fragment_login, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        if (model.getLoggedStatus()){
```

Comment:
This may be moving directly back to the list when this activity is reached via the back button. See my comment below about popping the navigation stack.